implementation/src/main/java/io/smallrye/graphql/scalar/time/DateCoercing.java

```diff
@@ -16,12 +16,10 @@
 public class DateCoercing implements Coercing {
 
     private final Class[] supportedTypes;
-    private final Converter converter;
     private final String name;
 
-    public DateCoercing(String name, Converter converter, Class... supportedTypes) {
+    public DateCoercing(String name, Class... supportedTypes) {
```

Comment:
Yea, this is here for future use, but we can add that back then. I was playing with fixing [#225](https://github.com/eclipse/microprofile-graphql/issues/225) in the spec.
At the moment Dates just fall through due to transformation, but if we change dates with transformation to be Strings, then this can create the correct date instance.

Reply:
I think it should be enough to change the type of formatted dates in the schema to string, because dates are formatted in any case (either in the format given with `@DateFormat or with ISO_TIME, ISO_DATE, etc).

Reply:
Tried this in https://github.com/ybroeker/smallrye-graphql/commit/06617144cc2d20faa585485dd15cd3b43b042e07, numbers and dates with `@DateFormat`-format (or equivalent) are included as Strings and are formatted correctly.

TCK currently fails because of eclipse/microprofile-graphql#225 and eclipse/microprofile-graphql#209, but once it is specified and tests are updated it should work.